content_processor.py

```diff
@@ -55,7 +55,7 @@ def preprocess(self, srctxt):
                 # print('tokenized sentence: ' + tokenized, flush=True)
                 segmented = self.bpe_source.process_line(tokenized)
             elif self.sp_processor_source:
-                print('raw sentence: ' + s, flush=True)
+                # print('raw sentence: ' + s, flush=True)
```

Comment:
Lets just remove it

Reply:
Pasha, this code has a lot of comments. This same command is already commented out 5 times. So I commented out for 'consistency'.